src/plugins/profiling/server/routes/load_topNStackTraces.ts

```diff
@@ -39,9 +41,9 @@ export function registerTraceEventsTopNStackTracesRoute(router: IRouter<DataRequ
         }),
       },
     },
-   async (ctx, request, response) => {
-      const timeFrom = parseInt(request.query.timeFrom);
-      const timeTo = parseInt(request.query.timeTo);
+    async (ctx, request, response) => {
+      const timeFrom = parseInt(request.query.timeFrom, 10);
```

Comment:
We don't need the radix (10) explicitly given here. If the first argument does not start with `0x`, radix 10 is assumed.

Reply:
The linter was not thinking the same...